error_test.go

```diff
@@ -95,21 +96,40 @@ func TestHandleError(t *testing.T) {
 	tests := map[string]struct {
 		resp    *http.Response
 		wantErr string
+		errors  Errors
 	}{
 		"single_error": {
 			resp: &http.Response{
 				Status: "400 " + http.StatusText(400),
 				Body:   ioutil.NopCloser(strings.NewReader(singleErrBody)),
 			},
-			wantErr: `400 Bad Request: ` + singleErrBody,
+			wantErr: `400 Bad Request: invalid record id`,
+			errors: Errors{
+				Error{
```

Comment:
I guess that's redundant:
```suggestion
				{
```